File: gp/src/main/java/uk/nhs/wales/gppractice/entity/GpPractice.java
package uk.nhs.wales.gppractice.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "gp_practice")
public class GpPractice {

    @Id
    @Column(name = "gp_practice_id")
    private String gpPracticeId;

    @Column(name = "lhb_code")
    private String lhbCode;

    public String getGpPracticeId() {
        return gpPracticeId;
    }

    public void setGpPracticeId(String gpPracticeId) {
        this.gpPracticeId = gpPracticeId;
    }

    public String getLHBCode() {
        return lhbCode;
    }
}

File: src/gp_practice_documents/entities/gp_practice.py
from sqlalchemy import Column, String

from .base import Base


class GpPractice(Base):
    """GP Practice entity."""

    __tablename__ = "gp_practice"

    gp_practice_id = Column(String, primary_key=True, name="gp_practice_id")
    lhb_code = Column(String, name="lhb_code")
